test/tests/ver-1ka/ver-1ka.i

```diff
@@ -0,0 +1,46 @@
+initial_pressure = 0 # Pa
+kb = 1.380649e-23 # Boltzmann constant J/K
+T = 500 # K
+S = 1e20 # 1/m^3/s
+V = 1 # m^3
+end_time = 10800 # s
```

Comment:
Are you sure to need such a small time step and such a large end time? 
The TMAP7 case goes to 10000 s.
I think a time step closer to 500 s should still give you very good match with the analytical solution. 

The reason why I am pointing this out is because your test currently takes a long time to run (> 300 s on the testing environment), when it should take under 2 seconds. Increasing the time step is a good way to reduce the computational time without (in that case) sacrificing accuracy.